sites/next.skeleton.dev/src/content/docs/components/listbox/svelte.mdx

```diff
@@ -0,0 +1,3 @@
+---
+layout: '@layouts/LayoutDoc.astro'
```

Comment:
I might recommend setting up at least one Svelte demo before switching over to React. You never know when we might run into an Astro quirk! Better to catch that sooner than later.

Reply:
Done! No quirks yet.